docs_user/modules/openstack-stop_openstack_services.adoc

```diff
@@ -108,7 +110,7 @@ for service in ${ServicesToStop[*]}; do
         if [ ! -z "${!SSH_CMD}" ]; then
             echo "Stopping the $service in controller $i"
             if ${!SSH_CMD} sudo systemctl is-active $service; then
-                ${!SSH_CMD} sudo systemctl stop $service
+                ${!SSH_CMD} sudo systemctl disable --now $service
```

Comment:
We don't want to disable services -- the control plane adoption rollback procedure (in case of problems with control plane adoption) needs to start them again, and we find which services we should start based on which services are enabled. Let's keep this one as it was.